### PR TITLE
docs: improve description of deploying-locally page

### PR DESCRIPTION
**Pull request by VMois**
- add some help with `reana-dev` for **conda** users
- add some help with `reana-dev git-fork for macOS users

### CODE REVIEW

File: docs/administration/deployment/deploying-locally/index.md [hunk 1]
```diff
@@ -2,7 +2,8 @@
 
 ## For researchers
 
-If you are a researcher and would like to try out deploying a small REANA cluster on your laptop, you can proceed as follows.
+If you are a researcher and would like to try out deploying a small REANA cluster on your laptop,
+you can proceed as follows.
```
Comment (mvidalgarcia) [inline docs/administration/deployment/deploying-locally/index.md:6]: Minor: What is the reasoning behind this change?

File: docs/administration/deployment/deploying-locally/index.md [hunk 2]
```diff
@@ -44,7 +45,11 @@ $ firefox https://localhost:30443
 
 If you are a developer and would like to work with REANA cluster source code, you can proceed as follows.
 
-Set up virtual environment and install `reana-dev` helper script:
+**1.** Find what Python version is currently used for development
+by checking a Python version in `Dockerfile` of one of the REANA cluster components
+([example](https://github.com/reanahub/reana-server/blob/master/Dockerfile)).
+Set up a virtual environment with the proper Python version
+and install the `reana-dev` helper script:
```
Comment (mvidalgarcia) [inline docs/administration/deployment/deploying-locally/index.md:52]:
```suggestion
**1.** Find what Python version is currently used for development
by checking the base image Python version in the `Dockerfile` of one of the REANA cluster components
(e.g. [REANA Server `Dockerfile`](https://github.com/reanahub/reana-server/blob/master/Dockerfile)).
Set up a virtual environment with the proper Python version
and install the `reana-dev` helper script:
```

File: docs/administration/deployment/deploying-locally/index.md [hunk 3]
```diff
@@ -53,36 +58,50 @@ $ source ~/.virtualenvs/reana/bin/activate
 (reana) $ pip install git+git://github.com/reanahub/reana.git#egg=reana
 ```
 
-Fork and clone all REANA source code repositories:
+In addition, for **conda** users, set `VIRTUAL_ENV` variable:
```
Comment (mvidalgarcia) [inline docs/administration/deployment/deploying-locally/index.md:61]:
```suggestion
In addition, for **conda** users, set `VIRTUAL_ENV` env variable:
```

File: docs/administration/deployment/deploying-locally/index.md [hunk 4]
```diff
@@ -53,36 +58,50 @@ $ source ~/.virtualenvs/reana/bin/activate
...
 (reana) $ reana-dev git-clone -c ALL -u johndoe
 ```
 
-Create a new REANA cluster:
+For **MacOS**, add `-b open` to `git-fork` command, example:
+
+```console
+(reana) $ reana-dev git-fork -c ALL -b open
+```
+
```
Comment (mvidalgarcia) [inline docs/administration/deployment/deploying-locally/index.md:80]: I'm not sure if we should include this or for something more generic like "You can customize the browser by adding the `-b` flag, e.g. macOS users can run ..."

Also, this command should be wrapped in the `eval` to open the browser directly, as it's above.

**Review by mvidalgarcia — COMMENTED** [review]
A couple of minor comments:
- Can you squash both commits in one?
- If you check the commit history, you'll realize we always have the commits prefixed. (e.g. `docs: foo bar`). I see you did it right for one of your commits, just a reminder :)
- Since you're at it, could you please update the [Copyright years](https://github.com/reanahub/docs.reana.io/blob/e6411382ea6076d21917b8a4cb91279fbb61ca46/mkdocs.yml#L11) on the footer?

File: docs/administration/deployment/deploying-locally/index.md [hunk 5]
```diff
@@ -53,36 +58,50 @@ $ source ~/.virtualenvs/reana/bin/activate
 (reana) $ pip install git+git://github.com/reanahub/reana.git#egg=reana
 ```
 
-Fork and clone all REANA source code repositories:
+In addition, for **conda** users, set `VIRTUAL_ENV` variable:
+
+```console
+(reana) $ export VIRTUAL_ENV=/../../miniconda3/envs/reana/bin/python
```
Comment (mvidalgarcia) [inline docs/administration/deployment/deploying-locally/index.md:64]: I cannot test the coda environment as I don't have it installed. Maybe @tiborsimko you can?

In any case, I might be wrong but the path doesn't look right to me. It starts from the root (`/`) and tries to go to the parent directory.
Plus, one shouldn't use relative paths in documentation as (I guess) the conda env dir location may vary depending on the user. Maybe we can use `~` instead? Not sure if `miniconda3` should be in the home directory.

File: docs/administration/deployment/deploying-locally/index.md (hunk 1 above)
Comment (VMois) [reply docs/administration/deployment/deploying-locally/index.md:6]: It didn't fit on my screen so I trailed it for better readability.

File: docs/administration/deployment/deploying-locally/index.md (hunk 5 above)
Comment (VMois) [reply docs/administration/deployment/deploying-locally/index.md:64]: This example is taken from my conda local installation => `/Users/vmois/miniconda3/envs/reana/bin/python`. I just made it more general. You can use relative, I guess, but I used the absolute path here.

Ohh. I just realized. I can replace with => `/<your_path_here>/miniconda3/envs/reana/bin/python`

Comment (VMois) [reply docs/administration/deployment/deploying-locally/index.md:64]: There is also `miniconda3`, `anaconda3`, and other similar names for different versions of conda. I am not sure how to show it here.

**Review by mvidalgarcia — APPROVED** [review]
Looks good 👍 

- Could you please amend the commit message to have `docs: ...` instead of `docs, ...`?
- Could you please rebase on top of master?

**Pull request merged**